voluncheer/opportunityboard/views/opportunityboard.py

```diff
@@ -1,27 +1,42 @@
+import json
+import logging
+
 from django.apps import apps
 from django.shortcuts import render
 
 from opportunityboard.models import Category
 from opportunityboard.models import Opportunity
 from opportunityboard.models import Subcategory
 from opportunityboard.models import Subsubcategory
+from opportunityboard.views.search import parse_search_filter
 
 Organization = apps.get_model("profiles", "Organization")
+OPPORTUNITY_PER_PAGE = 5
 
 
-def opportunityboard(request):
+def opportunityboard(request, page_number):
     """View for Opportunity Board"""
-    opportunity_lists = Opportunity.objects.order_by("-pubdate"[:20])
+    filter = parse_search_filter(request.GET)
+    logger = logging.getLogger(__name__)
+    logger.error(filter.gen_dict())
```

Comment:
We should not log errors during normal, successful operations. Polluting the error logs with non-error messages makes debugging more difficult.